idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -68,6 +69,13 @@
 DEFAULT_LOGPARAMS = {"sample_s3_output_path": SAMPLE_S3_OUTPUT_PATH,
                      "stats_file": os.path.join(RESULT_DIR, STATS_OUT)}
 
+# versioning
+## For now, index updates are infrequent and we can get their versions from S3.
+## If updates ever become frequent, we may want to check instead which version is actually on the
+## machine taking the job, possibly out of sync with the newest version in S3. 
+GSNAP_VERSION_FILE_S3 = 's3://czbiohub-infectious-disease/references/nt_k16.version.txt'
```

Comment:
I may have mentioned something to this effect before, but this isn't safe when multiple versions could be in use at the same time (even when transitioning between versions).